pkgs/development/python-modules/langgraph-runtime-inmem/default.nix

```diff
@@ -0,0 +1,61 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  hatchling,
+  blockbuster,
+  langgraph,
+  langgraph-checkpoint,
+  sse-starlette,
+  starlette,
+  structlog,
+}:
+
+buildPythonPackage rec {
+  pname = "langgraph-runtime-inmem";
+  version = "0.6.8";
+  pyproject = true;
+
+  # Not available in any repository
+  src = fetchPypi {
+    pname = "langgraph_runtime_inmem";
+    inherit version;
+    hash = "sha256-chPmwJ+tUJoRK5xX9+r6mbYf95ZbX4Z3mP6Ra19nBxM=";
+  };
+
+  # 1.5.24 (now 1.5.25) only affects non-cpython builds
+  # upgrade: https://github.com/NixOS/nixpkgs/pull/431547
+  postPatch = ''
+    substituteInPlace pyproject.toml \
+      --replace "blockbuster>=1.5.24,<2.0.0" \
+     "blockbuster>=1.5.23,<2.0.0"
+  '';
```

Comment:
Why not adding `blockbusters` to `pythonRelaxDeps`?

Reply:
What do you mean by "self-documenting"? Are you getting an error when using `pythonRelaxDeps = [ "blockbuster" ]`?

Reply:
The `substituteInPlace` shows exactly what constraint I'm relaxing, hence it's self-documenting. It's not clear what `pythonRelaxDeps` does to the dependency spec so I chose to narrow the change as much as possible.

I've been asked more frequently to change `pythonRelaxDeps` -> `substituteInPlace` (or a `.patch` file) more often than the other way around. Has something changed? 

I couldn't find any evidence of a treewide change except for #402181 (13 packages, with one that replaces `substituteInPlace` with `pythonRelaxDeps`) and no specific "use this and not that" notes in the documentation for this.

Reply:
Done.